MdeModulePkg/Universal/Variable/RuntimeDxe/VariableNonVolatile.c

```diff
@@ -6,6 +6,14 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
 
+#include <Library/UefiLib.h>

```

Comment:
Why do we need to add more headers here?

Reply:
Removed